src/scripts/main.js

```diff
@@ -2,4 +2,41 @@
 
 document.addEventListener('click', (e) => {
   // write code here
+  const wall = document.querySelector('.wall');
+  const spider = document.querySelector('.spider');
+
+  wall.addEventListener('click', () => {
```

Comment:
The click event listener for the 'wall' element is nested inside the document's click event listener. This means that every time the document is clicked, a new event listener is added to the 'wall'. This can lead to multiple event listeners being attached to the 'wall', causing the 'spider' to move multiple times for a single click. Consider moving the 'wall' event listener outside of the document's click event listener.